app/views/providers/contact.html.erb

```diff
@@ -56,6 +56,13 @@
         <%= f.text_field field, options.merge(class: 'govuk-input govuk-!-width-two-thirds') %>
       <% end %>
 
+      <% if @provider.provider_type == "lead_school" %>
+        <%= render "shared/form_field",
+          form: f, field: :urn, label: "URN (Unique Reference Number)" do |field, options| %>
+          <%= f.text_field field, options %>
```

Comment:
Can we have the same size text box as UKPRN, which will be above it please?

<%= f.text_field field, options.merge(class: 'govuk-input govuk-input--width-10') %>

Reply:
all done 👍